### PR TITLE
Pull Request template

### PR DESCRIPTION
**Pull request by ildyria**
Just a nice text to be placed here. :)

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<!--
+Thank you for contributing to Lychee! We only accept PR to the master branch.
+
+In addition, please describe the benefit to end users; the reasons it does not break any existing features, etc.
+
+If you're pushing a Feature:
+- Title it: "This new feature"
+- Describe what the new feature enables
+- Ensure it doesn't break any feature.
```
Comment (qwerty287) [inline .github/PULL_REQUEST_TEMPLATE.md]: I'd add a note about tests:
```suggestion
- Ensure it doesn't break any feature.
- Add tests to test the feature.
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+If you're pushing a Fix:
+- Title it: "Fixes the bug name"
+- If it is a fix an an existing issue start the description with `fixes #xxxx` where xxxx is the issue number.
+- Describe how it fixes in a few words.
+- Ensure it doesn't break any feature.
```
Comment (qwerty287) [inline .github/PULL_REQUEST_TEMPLATE.md:17]: I'd add a note about tests:
```suggestion
- Ensure it doesn't break any feature.
- Add a test that triggered the bug before fixing it.
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+All Pull Requests run with extensive tests for stable and latest versions of PHP. 
+Ensure your tests pass or your PR may be taken down.
+
+Don't worry if your code styling isn't perfect! php-cs-fixer will automatically create a pull request with any style fixes. This allows us to focus on the content of the contribution and not the code style.
```
Comment (kamil4) [inline .github/PULL_REQUEST_TEMPLATE.md:24]: Shouldn't we at least mention `make formatting` and `make phpstan` here? They run just fine locally without any undue burdens (such as the empty DB requirement of `make test`). Sure, it's nice that we have the online fallback for `php-cs-fixer` but I would still stress that it's better to run them locally first before opening a PR...

Comment (ildyria) [reply .github/PULL_REQUEST_TEMPLATE.md:24]: Better ?

Comment (kamil4) [reply .github/PULL_REQUEST_TEMPLATE.md:24]: Yep...

**Review by kamil4 — APPROVED** [review]

**Comment by qwerty287** [comment]
@ildyria you removed the test notices again.

**Comment by ildyria** [comment]
No, tests are not run when the modified files are just .md

**Comment by qwerty287** [comment]
No, I mean that you accidentally removed the lines I suggested and you added before with [this commit](https://github.com/LycheeOrg/Lychee/pull/1485/commits/05c90d065c0dbefb9a2661165227fe07c303a4c1).

**Comment by ildyria** [comment]
They are still there, I just moved them up in the lists. :)

**Review by qwerty287 — APPROVED** [review]
OK, sorry.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
+<!--
+Thank you for contributing to Lychee! We only accept PR to the master branch.
+
+In addition, please describe the benefit to end users; the reasons it does not break any existing features, etc.
+
+If you're pushing a Feature:
+- Title it: "This new feature"
+- Describe what the new feature enables
+- Add tests to test the new feature.
+- Ensure it doesn't break any feature.
```
Comment (d7415) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
- Ensure it doesn't break any existing features.
```
?

**Pull request merged**